src/guides/v2.3/config-guide/redis/config-redis.md

```diff
@@ -27,6 +27,37 @@ Installing and configuring the Redis software is beyond the scope of this guide.
 *  [digitalocean](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-redis)
 *  [Redis documentation page](http://redis.io/documentation)
 
+## Set up redis configuration {#config-redis-setup}
+
+Depending on your installation, you can usually find your Redis configuration in one of the following files: `/etc/redis/redis.conf` or `/etc/redis/<port>.conf`
+
+To optimize the Redis instance for your requirements, you get best results by using a dedicated instance for each session, Magento cache and FPC.
+
+For sessions, we recommend that you enable persistence to copy Redis data to disk using either of the following persistence options: regular Redis Database Backup (RDB) snapshots, or Append Only File (AOF) persistence logs.
+
+*  RDB (Redis Database Backup) snapshots store the complete database in a dump file after a given time, when a minimum number of keys have changed since the last save. Use the `save` setting inside the `redis.conf` file to configure this setting.
+
+*  AOF (Append Only File) stores each write operation sent to Redis in a journal file. Redis reads this file on restart only and uses it to restore the original dataset.
+
+You can also enable both the RDB and AOF options at the same time. For additional details including the advantages and disadvantages of the persistence options, see the [Redis Persistence documentation](https://redis.io/topics/persistence).
+
+For the cache instance, set up the instance so that it is large enough to store your entire Magento cache.
+Size requirements depend on different factors like the number of products and store views, but the size requirement for the the file system cache gives you the order of magnitude. Persistence is not required for the cache instance because the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).
```

Comment:
@jonashrem  Can you provide an example for the cache instance size?

Reply:
@meker12 I just made some changes to the sentence. Is this clear now?

Reply:
>  Size requirements depend on different factors like the number of products and store views. When you have your Magento Cache previously put into your file system, you can check out the size of the cache folder for a good starting point to set up your Redis instance. For example, when you have a 5 GB 'var/cache' folder your Redis instance should have a similar size available. Persistence is not required for the cache instance because the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).